src/rockstor/system/services.py

```diff
@@ -192,8 +196,10 @@ def service_status(service_name, config=None):
     elif service_name == "sftp":
         # Delegate sshd's sftp subsystem status check to system.ssh.py call.
         return is_sftp_running(return_boolean=False)
-    elif service_name in ("replication", "data-collector", "ztask-daemon"):
```

Comment:
Take care with the ztask-daemon service re-name, I seem to remember there were DB elements sill using this old name which I left as-was (with ztask to huey move) due to the depth of the changes required. But maybe it's better we get this name-change over and done with. Especially given your move to a generic name now. My concern revolves around DB names within our services model. Again apologies for just nipping in on a draft here.